File: URI/1 - INICIANTE/C/1173 - PreenchimentoDeVetor1.c
#include <stdio.h>

int main(){
	int x = 0, num, vetor[10];
	scanf(" %d", &num);
	vetor[0] = num;
	for(x = 1; x < 10; x++){
		vetor[x] = vetor[x-1]*2;
	}
	for(x = 0; x < 10; x++){
		printf("N[%d] = %d\n", x, vetor[x]);
	}
	return 0;
}

File: URI/1 - INICIANTE/Python/1173 - PreenchimentoDeVetor1.py
num = int(input())
vetor = []
vetor.append(num)
for y in range(1, 10):
    temp = vetor[y-1]*2
    vetor.append(temp)
for y in range(0, 10):
    print('N[{}] = {}'.format(y, vetor[y]))